src/tests/JIT/HardwareIntrinsics/Arm/Shared/SveConditionalSelect.template

```diff
@@ -0,0 +1,362 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
```

Comment:
Is this template based on an existing template?

Can we just use a generic 3 op template here? I don't think I see anthing specific to conditionalselect